dev/io.openliberty.security.oidcclientcore.internal/test/io/openliberty/security/oidcclientcore/authentication/JakartaOidcAuthorizationRequestTest.java

```diff
@@ -266,6 +265,35 @@ public void test_addConditionalParameters_multipleParameters() throws OidcDiscov
         authzRequest.addExtraParameters(authzParameters);
     }
 
+    @Test
+    public void test_addConditionalParameters_twoEqualsSigns() throws OidcDiscoveryException {
+        String key = "some_parameter";
+        String value = "value=1";
+        final String[] extraParams = new String[] { key + "=" + value };
+        mockery.checking(new Expectations() {
+            {
+                one(config).getExtraParameters();
+                will(returnValue(extraParams));
+                one(authzParameters).addParameter(key, value);
+            }
+        });
+        authzRequest.addExtraParameters(authzParameters);
+    }
+
+    @Test
+    public void test_addConditionalParameters_noEqualsSign() throws OidcDiscoveryException {
```

Comment:
This is actually testing the same thing as `test_addConditionalParameters_oneParameter_keyWithNoValue`.